server/errors.json

```diff
@@ -1238,5 +1238,15 @@
     "help": "",
     "url": "",
     "deprecates": ""
+  },
+  {
+    "constant": "JSConsumerMaxRequestBatchRequired",
+    "code": 400,
+    "error_code": 10126,
+    "description": "consumer max request batch needs to be specified",
```

Comment:
Should we include the current limit here? Something like: `consumer max request batch needs to be specified and cannot be more than {limit}`. If so, re-run `go generate` and then you will be able to pass the limit in the NewJSConsumerMaxRequestBatchRequiredError() function.

Reply:
done